client/actions/tests/agenda_test.js

```diff
@@ -113,7 +113,7 @@ describe('agenda', () => {
                     expect(apiSpy.save.args[0]).toEqual([{}, item])
                     expect(notify.success.args[0]).toEqual(['The agenda has been created/updated.'])
 
-                    expect(dispatch.callCount).toBe(10)
+                    expect(dispatch.callCount).toBe(11)
```

Comment:
this kind of test doesn't mean much to me. I'm used to removed them when they start to fail.
Unless you convince me to the other way, I will continue to do it, and won't blame you if do it also ;)